start.ipynb

```diff
@@ -15,6 +15,25 @@
     "}"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "html"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "%%html\n",
+    "\n",
+    "<style>\n",
+    "  .output_subarea {\n",
+    "    max-width: none !important;\n",
+    "  }\n",
```

Comment:
I am going to trust you on this one. :sweat_smile:

Reply:
Okay, I guess this is not that complicated, you're just ensuring that the `max-width` parameter is not set.